setup.py

```diff
@@ -0,0 +1,37 @@
+from setuptools import setup
+
+setup(
+    name='trafficcontrol',
+    version='1.0.0',
+    long_description=open('README.md').read(),
+    long_description_content_type='text/markdown',
+    py_modules= [],
+    install_requires=[
+        'Apache-TrafficControl @ file:///os.path.abspath(os.getcwd())/traffic_control/clients/python',
```

Comment:
I think we should use Apache-TrafficControl==3.1.0 here

Reply:
Sure I'll take a look into it and change